arch/x86/core/ia32/gdbstub.c

```diff
@@ -177,8 +177,6 @@ size_t arch_gdb_reg_readone(struct gdb_ctx *ctx, uint8_t *buf, size_t buflen,
 		if (buflen >= 2) {
```

Comment:
isn't this branch always true as well ?  Because of this check `if (buflen < (sizeof(unsigned int) * 2)) `  buflen will be greater than 2 in this else statement.

Reply:
Yep... fixed.